holmes.go

```diff
@@ -213,7 +213,15 @@ func (h *Holmes) startDumpLoop() {
 
 // goroutine start.
 func (h *Holmes) goroutineCheckAndDump(gNum int) {
-	if !h.opts.GrOpts.Enable {
+	// get a copy instead of locking it
+	coolDown := h.opts.CoolDown
+
+	grOpts, exist := h.opts.GetGrOpts()
```

Comment:
won't be non-exists? unless it's an invalid usage?

Reply:
yes, it's an outdated function, it won't be used.
Deleted.